### ISSUE TITLE
Unsupported major.minor version 52.0

### ISSUE THREAD

**Issue by humandb** [adamw/elasticmq#57]
Downloading `https://s3-eu-west-1.amazonaws.com/softwaremill-public/elasticmq-server-0.8.10.jar` and running 

```
java -jar elasticmq-server-0.8.10.jar
```

results in the following error

```
java.lang.UnsupportedClassVersionError: com/typesafe/config/ConfigFactory : Unsupported major.minor version 52.0
	at java.lang.ClassLoader.defineClass1(Native Method) ~[na:1.7.0_75]
	at java.lang.ClassLoader.defineClass(ClassLoader.java:800) ~[na:1.7.0_75]
	at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142) ~[na:1.7.0_75]
	at java.net.URLClassLoader.defineClass(URLClassLoader.java:449) ~[na:1.7.0_75]
	at java.net.URLClassLoader.access$100(URLClassLoader.java:71) ~[na:1.7.0_75]
	at java.net.URLClassLoader$1.run(URLClassLoader.java:361) ~[na:1.7.0_75]
	at java.net.URLClassLoader$1.run(URLClassLoader.java:355) ~[na:1.7.0_75]
	at java.security.AccessController.doPrivileged(Native Method) ~[na:1.7.0_75]
	at java.net.URLClassLoader.findClass(URLClassLoader.java:354) ~[na:1.7.0_75]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:425) ~[na:1.7.0_75]
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:308) ~[na:1.7.0_75]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:358) ~[na:1.7.0_75]
	at org.elasticmq.server.Main$.main(Main.scala:15) ~[elasticmq-server-0.8.10.jar:0.8.10]
	at org.elasticmq.server.Main.main(Main.scala) ~[elasticmq-server-0.8.10.jar:0.8.10]
```

It seems like Config only supports Java 8 from version 1.3.0 (https://github.com/typesafehub/config/issues/321). I think you'll need to change the README.me file as it says it's compatible with Java 6 (I'm currently running Java 7).

**Comment by adamw** [comment]
That's a mistake on my part - I released 0.8.11 which uses config 1.2.1, should work on java 6. Sorry!

**Comment by humandb** [comment]
No worries. Thanks!
